Gemfile.lock

```diff
@@ -44,35 +44,35 @@ GEM
     loofah (2.4.0)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
-    method_source (0.9.2)
+    method_source (1.0.0)
     mini_portile2 (2.4.0)
-    minitest (5.13.0)
+    minitest (5.14.0)
     money (6.13.6)
       i18n (>= 0.6.4, <= 2)
-    nokogiri (1.10.8)
+    nokogiri (1.10.9)
       mini_portile2 (~> 2.4.0)
     pagerduty (2.1.2)
       json (>= 1.7.7)
-    rack (2.0.8)
+    rack (2.2.2)
     rack-test (1.1.0)
       rack (>= 1.0, < 3)
     rails-dom-testing (2.0.3)
       activesupport (>= 4.2.0)
       nokogiri (>= 1.6)
     rails-html-sanitizer (1.3.0)
       loofah (~> 2.3)
-    railties (6.0.2.1)
-      actionpack (= 6.0.2.1)
-      activesupport (= 6.0.2.1)
+    railties (6.0.2.2)
```

Comment:
### [railties](https://rubyonrails.org)



[_[change-log](https://github.com/rails/rails/blob/v6.0.2.2/railties/CHANGELOG.md), [source-code](https://github.com/rails/rails/tree/v6.0.2.2/railties)_]





<details>
<summary>Commits</summary>

A change of **3** commits. See the full changes on [the compare page](https://github.com/rails/rails/compare/v6.0.2.1...v6.0.2.2).

These are the individual commits:
- (1251d88) [Fix possible XSS vector in JS escape helper](https://github.com/rails/rails/commit/1251d8817264744163fb12a3dba05ce61be5371b)
- (fa413f2) [update version](https://github.com/rails/rails/commit/fa413f2e8411cb22f6d76b119a79a25aab1e935c)
- (157920a) [Preparing for 6.0.2.2 release](https://github.com/rails/rails/commit/157920aead96865e3135f496c09ace607d5620dc)


</details>